.github/workflows/require-labels.yml

```
@@ -0,0 +1,20 @@
+name: Require PR Label
+
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize, reopened, edited]
+
+jobs:
+  require-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for required label
+        uses: actions/github-script@v7
```

Can we use the commit SHA here?
```suggestion
        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8.0.0
```